docs/src/tutorial/external.rst

```diff
@@ -15,7 +18,17 @@ For example, let's say you need a low-level way to parse a number from
 a ``char*`` value.  You could use the ``atoi()`` function, as defined
```

Comment:
In pure python mode, you need to use `cython.p_char`

Reply:
I understand the text in a way that it discuss in context of C language (e.g. it mentions also `stdlib.h` header file.